cv32e40x_manifest.flist

```diff
@@ -29,6 +29,7 @@
 +incdir+${DESIGN_RTL_DIR}/../sva
 
 ${DESIGN_RTL_DIR}/include/cv32e40x_pkg.sv
+${DESIGN_RTL_DIR}/include/cv32e40x_sequencer_pkg.sv
```

Comment:
I don't think we need a new file. Why not just use the existing package file?

Reply:
Agreed, merging this into the existing package file.